.github/CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 <h3>New features since last release</h3>
 
+* Adds a device test suite, located at `pennylane/plugins/tests`, which can be used 
+  to run generic tests on core or external devices calling 
+  
+  >>> pytest pennylane/plugins/tests --device default.qubit --shots 1234 --analytic False                                                                                                                                                                                                                                                                    
+  
+  The command line arguments are optional.
+   
+  * If `--device` is not given, the tests are run on the core devices that ship with PennyLane. 
+  
+  * If `--shots` is not given, a default of 50000 is used.
+  
+  * If `--analytic` is not given, the device's default is used.
+  
+  Other arguments of the device, such as `strawberryfields.fock`'s compulsory `cutoff_dim`, 
```

Comment:
(Minor suggestion): just to have a qubit model example for this CHANGELOG entry (as CV would be upcoming)
```suggestion
  Other arguments of the device, such as `qiskit.aer`'s compulsory `backend_options`, 
```